camel-k-loader-yaml/camel-k-loader-yaml/src/test/resources/routes/RoutesTest_aggregator.yaml

```diff
@@ -19,7 +19,7 @@
     steps:
       - aggregate:
           strategy-ref: "aggregatorStrategy"
-          completion-size: 2
+          completion-size: "2"
```

Comment:
why is it changed from a number to a string? is it a bug/limitation in the schema? or is it the example which had a bug?

Reply:
it is because the completion size is a string field: https://github.com/apache/camel/blob/c2f5ba87c4173e36e99f6332e44f055572e72d24/core/camel-core-engine/src/main/java/org/apache/camel/model/AggregateDefinition.java#L97

camel does not care much as it is able to convert among types but the type is of course relevant for the schema validation

Reply:
if needed, it seems to be possible to use multiple primitive type for the schema definition: https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.7.6.1